pkg/util/util.go

```diff
@@ -62,3 +66,14 @@ func ParseEnv(p string, fs *flag.FlagSet) error {
 
 	return err
 }
+
+func AddAliases(fs *flag.FlagSet, aliases map[string][]string, descSuffix string) {
```

Comment:
Is this only used in tests? If yes, lets move to the tests file.

Reply:
No, please check again, its used in `pkg/runner/flags.go`.